scripts/sct_compute_mtr.py

```diff
@@ -56,52 +56,37 @@ def main(args=None):
 
     fname_mt0 = arguments['-mt0']
     fname_mt1 = arguments['-mt1']
-    remove_temp_files = int(arguments['-r'])
+    fname_mtr = arguments['-o']
+    file_out = os.path.basename(fname_mtr)[:os.path.basename(fname_mtr).rfind('.nii')]
```

Comment:
why do you need file_out and ext_out? just use fname_mtr to write your output file

Reply:
ok got it! Fixed in commit [f49fe06](https://github.com/YangHee-Min/spinalcordtoolbox/commit/f49fe066999249401f2ca4e9692792f8472b72c3)